jib-cli/src/main/java/com/google/cloud/tools/jib/cli/war/WarFiles.java

```diff
@@ -81,7 +81,10 @@ private static List<String> computeEntrypoint(
       return entrypoint;
     }
     if (commonContainerConfigCliOptions.isJettyBaseimage()) {
-      return ImmutableList.of("java", "-jar", "/usr/local/jetty/start.jar");
+      // jetty 12+ requires explicitly enabling ee10-deploy module. See:
+      // https://eclipse.dev/jetty/documentation/jetty-12/operations-guide/index.html
+      return ImmutableList.of(
```

Comment:
Note: we're pulling in "jetty" without a version, currently that means it's pulling v12 which starts up without serving content by default. This flag makes it serve content like in versions prior to v12